Curtains/Curtains.Application/CurtainsService/MarketingInfoService.cs

```diff
@@ -0,0 +1,55 @@
+using Curtains.Application.DTO;
+using Curtains.Infrastructure.Interfaces;
+using AutoMapper;
+using Microsoft.Extensions.Logging;
+using Curtains.Infrastructure.Shared.Exceptions;
+using Curtains.Application.CurtainsService.Interfaces;
+
+namespace Curtains.Application.CurtainsService
+{
+    /// <summary>
+    /// Class <c> OurWorksService </c> describes the interaction model of the data transfer object to the database.
```

Comment:
Please  previously look what you ctrl+C -> ctrl+V

Reply:
done!!!!